scripts/check-all-the-things.ps1

```diff
@@ -0,0 +1,11 @@
+# In theory we could write all of this with Bash style &&,
+# but in order to maintain backwards compatibility
+# with Powershell <7.0 (what most Windows users have preinstalled)
+# we write it in the old-school style
+lerna clean --yes
+lerna run clean
+if ($?) { lerna bootstrap }
+if ($?) { lerna run compile }
+if ($?) { lerna run fix-lint }
+if ($?) { lerna run lint }
+if ($?) { lerna run test }
```

Comment:
What does $? mean? Does it contain the result of the previous command?

Reply:
yep